features/step_definitions/navigation_steps.rb

```diff
@@ -106,6 +106,10 @@
   expect(page.current_path).to eq(profile_path)
 end
 
+Then(/^I should be on the admin auction page for that auction$/) do
```

Comment:
Actually, this is correct. IT's the "check I'm in the right place" equivalent to the "open the URL" for the admin auction show page